reference_guide/ui_themes/themes_customize.md

```diff
@@ -166,6 +178,49 @@ The following example sets the background color for all labels to the color #F6E
 ```
 The `Label.background` entry supersedes, in the narrower context of label backgrounds, any default color as well as any wildcard color assigned to backgrounds.
 
+#### Customizing the Color of UI Tabs
+The color of UI Tabs are changed by adding a key-value pair to the `"ui": {}` section of a Theme description file.
+The `key` is the `element.property` format and the `value` is the custom color.
+
+There are two implementations of tabs in the IntelliJ Platform:
+* Editor Tabs, which e.g., represent open files in the [Editor window](https://www.jetbrains.com/help/idea/using-code-editor.html), and in [Tool Window bars](https://www.jetbrains.com/help/idea/tool-windows.html#bars_and_buttons).
+* Tabbed Panes, which e.g., are used for the [Run/Debug Configurations dialog](https://www.jetbrains.com/help/idea/run-debug-configurations-dialog.html).
+
+The control keys for UI Tabs were expanded from build 191 to 192 of the IntelliJ Platform.
+The build 191 control keys are compatible with build 192 and later versions of the IntelliJ Platform. 
+
+| Build 191 Element | Build 192 Element | Description of 192 Element |
+|------|---------|---------|
+| N/A  | **`DefaultTabs`** | Applied to all tabs except `TabbedPane`, _unless_ overridden by a more specific Tab control element. This element is the background under all tabs. |
+| **`EditorTabs`**  | **`EditorTabs`** | Applied only to Editor tabs. Overrides any `DefaultTab` settings. 192 has many more `property` settings than 191. |
+| **`ToolWindow.HeaderTab`** | **`ToolWindow`** | Applied only to Tool Window tabs. Overrides any `DefaultTab` settings. 192 has many more `property` settings than 191. |
+| **`DebuggerTabs`** | **`DefaultTabs`** | `DefaultTab` settings are used instead of `DebuggerTabs`, except for key `DebuggerTabs.underlineHeight`. |
+| N/A | **`TabbedPane`** | New in 192. Applied only to Tabbed Panes. |
+| **`Plugins.Tab`** | **`TabbedPane`** | Use `TabbedPane` instead. |
+| **`SearchEverywhere.Tab`** | **`SearchEverywhere.Tab`** | No change. |
+
+Methods for identifying UI control keys are in the [Finding Attribute Keys for UI Controls](#finding-attribute-keys-for-ui-controls) section. 
```

Comment:
The full list of color keys can be found in these files:
 https://github.com/JetBrains/intellij-community/tree/master/platform/platform-resources/src/themes/metadata. Could they be mentioned in the "Finding Attribute Keys" section?

Reply:
please no. theme authors _must_ rely on what autocompletion gives them as it depends on the current platform version (which is not necessarily the same as link above)

Reply:
Platform version — agree. It's that metadata files are much more convenient to read when you need to learn what keys are there and what descriptions and state (e.g. deprecated) they have. Completion can show documentation but I doubt many people know that. Anyway, I don't have a strong opinion that metadata files should be mentioned, just a suggestion.